pkg/virt-handler/cgroup/cgroup_v2_manager.go

```diff
@@ -71,7 +73,17 @@ func (v *v2Manager) Set(r *runc_configs.Resources) error {
 		log.Log.V(5).Infof("cgroupsv2 device allowlist: rule after appending current+new: type: %d permissions: %s allow: %t major: %d minor: %d", rule.Type, rule.Permissions, rule.Allow, rule.Major, rule.Minor)
 	}
 
-	return v.execVirtChroot(&resourcesToSet, map[string]string{"": v.dirPath}, v.isRootless, v.GetCgroupVersion())
+	subsystemPaths := map[string]string{
+		"target": v.dirPath,
+	}
+	if targetDir, parentPath := filepath.Base(v.dirPath), path.Dir(v.dirPath); targetDir == "container" && strings.HasSuffix(parentPath, ".scope") {
+		// This is needed for crun based installations for a brief period of time
+		// crun will eventually stop configuring both cgroups
+		subsystemPaths["parent"] = parentPath
+	}
```

Comment:
Can you please add another log entry under this `if` without special verbosity?
I think it's valuable so we'd know when this is being used, and when we'd be able to remove this code in the future

Reply:
my impression is that we're already adding too much logging, why doesn't the V(5) log below suffice?
you could always tweak the verbosity on any kv platform
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/52ddda679e2f09fb4c89fece4d718e5ac577d30e/docs/cluster-configuration.md?plain=1#L926

Reply:
I was thinking that this way we can check logs of e.g. periodic lanes and see if this code is being used or not.
But it's not a hard blocker from my side or anything, feel free to ignore.

Is there some kind of ETA from crun's side to when we won't need this anymore?
Is there a good way to track this?

Reply:
You're right, we should be doing verbosity=5 for CI. Easier to debug. I played with it before:
https://github.com/kubevirt/kubevirt/pull/13221
Would you consider something like this?

Reply:
Yup, makes a lot of sense IMO.
If you want to do that in a follow-up it's even better than my log entry suggestion :)